config/csm/csm_config.yaml

```diff
@@ -93,6 +93,7 @@ Restcall_LC:
   csmuser_permission_endpoint: "/api/v2/permissions"
   bucket_policy_endpoint : "/api/v2/s3/bucket_policy/{}"
   perf_stats_endpoint: "/api/v2/metrics/perf"
+  get_system_topology: "/api/v2/system/topology"
```

Comment:
please add "_endpoint" as there is a function with same name

Reply:
we dont need GET -- it should be system_topology

Reply:
changed the endpoint to system_topology_endpoint